src/test/java/com.careem.opensource/ParserTest.java

```diff
@@ -13,39 +13,45 @@
 
 public class ParserTest {
 
+  // TODO: 9/12/18  should we convert these into final ?
+
   private static Parser parser;
-  private static String chunk;
+  private static String chunkContainsEvacuationTime;
   private static String chunkDoesNotContainPause;
   private static String chunkContainEvacuationPause;
   private static String emptyChunk;
+  private static String chunkContainsCouncurrentMark;
```

Comment:
why did you make all these fields static? these are test variables  that you define, not constant shared variables